config/default.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"services": {
 		"calendar": "http://localhost:3000",
-		"content": "https://schul-cloud.org:8090",
+		"content": "https://schul-cloud.org:4040",
```

Comment:
@kremer-io das wird in Prod eh überschrieben, richtig?

Reply:
joa, ist aber auch das selbe ... ist einfach nur dafür, das es bei den Devs lokal nicht immer crasht.
Vielleicht ne Beta Instanz Mal aufsetzen?